package.json

```diff
@@ -12,7 +12,8 @@
     "lint:fix": "eslint . --fix",
     "test": "vitest",
     "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "start": "npm run dev"
```

Comment:
Do we need this? I'm not sure this command makes starting the app faster and I would suggest starting the app in the production mode while using the `start` command.

For now I would leave only the `dev` command and then add the `start` command if the app is ready for production.

You can simply start the dev server using `yarn dev` command which is shorter than `yarn start` or `npm start`.